### ISSUE TITLE
run internal kubectl when os.Args[0] is 'kubectl'

### ISSUE THREAD

**Issue by drewpca** [kubernetes/minikube#8857]
It would be nice if this worked:

```
% sudo install minikube-linux-amd64 /usr/local/bin/minikube
% sudo ln -s /usr/local/bin/minikube /usr/local/bin/kubectl
% kubectl version
```
No harm to existing workflows (as long as we don't force the creation of that symlink-- see https://github.com/rancher/k3s/issues/521), and it would save on aliases and '--' argument separators.

Related: https://github.com/kubernetes/minikube/issues/4684
Related: https://github.com/kubernetes/minikube/issues/4697

**Comment by afbjorklund** [comment]
I like this suggestion, we talked about our solutions like providing an "--install" flag or echoing the program path.

**Comment by kadern0** [comment]
@sharifelgamal this has been fixed with #8872

**Comment by sharifelgamal** [comment]
Excellent, thank you!
